src/optim/FishLeg/fishleg.py

```diff
@@ -105,40 +112,35 @@ def __init__(
         draw: Callable[[nn.Module, torch.Tensor], Tuple[torch.Tensor, torch.Tensor]],
         nll: Callable[[nn.Module, Tuple[torch.Tensor, torch.Tensor]], torch.Tensor],
         aux_dataloader: torch.utils.data.DataLoader,
-        lr: float = 1e-2,
-        eps: float = 1e-4,
+        likelihood: FishLikelihood,
+        fish_lr: float = 5e-2,
         weight_decay: float = 1e-5,
         beta: float = 0.9,
-        update_aux_every: int = -3,
-        aux_lr: float = 1e-3,
+        update_aux_every: int = 10,
+        aux_lr: float = 1e-4,
         aux_betas: Tuple[float, float] = (0.9, 0.999),
         aux_eps: float = 1e-8,
         damping: float = 1e-5,
         pre_aux_training: int = 10,
         differentiable: bool = False,
         sgd_lr: float = 1e-2,
+        initialization: str = 'uniform',
         device: str = "cpu",
+        num_steps = None
     ) -> None:
         self.model = model
-        self.plus_model = copy.deepcopy(self.model)
-        self.minus_model = copy.deepcopy(self.model)
         self.sgd_lr = sgd_lr
-        self.lr = lr
+        self.fish_lr = fish_lr
```

Comment:
I think it would be best to stick to the hyperparameter names set out in the paper. But happy to be overruled on this.